OCLB.user.js

```diff
@@ -154,6 +154,7 @@ addJS(function () {
       'vibrantsnow',
       'wytherwing',
       'zestylimey',
+      'ZjeroXytz',
```

Comment:
might need lowercasing to match
```suggestion
      'zjeroxytz',
```